src/modality-specific-files/positron-emission-tomography.md

```diff
@@ -129,8 +162,10 @@ the same [`task-<label>`](../appendices/entities.md#task) entity SHOULD be used.
 For further details, see
 [Task (including resting state) imaging data](./magnetic-resonance-imaging-data.md#task-including-resting-state-imaging-data).
 
+<!--
 In addition to the imaging data (`*.nii`) a `_pet.json` sidecar file MUST be provided.
 The included metadata are divided into sections described below.
+-->
```

Comment:
Commented this paragraph, as it appears in incorrect section (JSON must be present no matter if it's pure PET of PET-MRI dataset). Also, the same information appears directly below.